code/controllers/subsystem/vote.dm

```diff
@@ -226,9 +226,9 @@ SUBSYSTEM_DEF(vote)
 					to_chat(usr, "<span class='warning'>The next ship map has already been selected.</span>")
 					return FALSE
 				var/list/maps = list()
+				if(!config.maplist)
+					return
 				for(var/map in config.maplist[SHIP_MAP])
-					if(!config.maplist)
-						return
```

Comment:
Ok so, i did a mistake in my other PR, it was on the wrong lines. 
This prevents runtime in integration test